.github/workflows/unit-tests.yaml

```diff
@@ -0,0 +1,30 @@
+name: unit-tests-boa
+
+on: ["push", "pull_request"]
```

Comment:
just pick either push or pull_request, using both generates twice the same job for every commit in a PR

Reply:
![image](https://github.com/curvefi/twocrypto-ng/assets/2369243/923f33db-f988-439f-86d6-f13c3b23c3aa)